packages/fast-jss-manager-angular/README.md

```diff
@@ -0,0 +1,2 @@
+
+An angular directive and component library for managing component JSS stylesheets.
```

Comment:
Angular should be capitalized in this sentence. Additionally, we should describe how to configure, install, and use this package.